src/controller.c

```diff
@@ -47,16 +47,16 @@ void act_on_user_input(char user_input, State* s,
             *frame_counter += frames_until_next_move;
             break;
         case LEFT_KEY: // move block left
-            move_block(s, -1);
+            move_block(s, -1, 0);
```

Comment:
can you declare this as an enum in `utils.h` so that {-1,0,1} becomes {LEFT, NO_ROTATE, RIGHT} or something like that